lib/Echidna/Exec.hs

```diff
@@ -97,10 +97,14 @@ usingCoverage cov = maybe (cov >> liftSH exec1 >> usingCoverage cov) pure =<< us
 coveragePoints :: CoverageMap -> Int
 coveragePoints = sum . fmap S.size
 
+-- | Given good point coverage, count simplified unique points (to report).
+scoveragePoints :: CoverageMap -> Int
+scoveragePoints = sum . fmap (S.size . S.map fst)
```

Comment:
maybe we want to add a comment here about how it disregards `TxResults` as a set as it is a bit unclear

Reply:
Done.